lib/music_importer.rb

```diff
@@ -0,0 +1,16 @@
+# MusicImporter class that imports a song from a file
+class MusicImporter
+  attr_accessor(:path)
```

Comment:
You have the option to remove parenthesis. I think it's more expressive if you do following popular convention mostly in rails.
```ruby
attr_accessor :path
```